docs/es-modules/profiles.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Cloudinary Video Player</title>
+    <link
+      href="https://res.cloudinary.com/cloudinary-marketing/image/upload/f_auto,q_auto/c_scale,w_32,e_hue:290/creative_staging/cloudinary_internal/Website/Brand%20Updates/Favicon/cloudinary_web_favicon_192x192.png"
+      rel="icon"
+      type="image/png"
+    />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  </head>
+  <body>
+    <div class="container p-4 col-12 col-md-9 col-xl-6">
+      <nav class="nav mb-2">
+        <a href="/index.html">&#60;&#60; Back to examples index</a>
+      </nav>
+      <h1>Cloudinary Video Player</h1>
+      <h3 class="mb-4">Profiles</h3>
+
+      <h5>Player with default profile</h5>
+      <video
+        id="player-default-profile"
+        class="cld-video-player"
+        width="500"
+        playsinline
+        controls
+        autoplay
+        muted
+      ></video>
+
+      <br /><br />
+      <h5>Player with custom (blur) profile</h5>
+      <video
+        id="player-custom-profile"
+        class="cld-video-player"
+        width="500"
+        playsinline
+        controls
+        autoplay
+        muted
+      ></video>
+
+      <p class="mt-4">
+        <a href="https://cloudinary.com/documentation/cloudinary_video_player"
+          >Full documentation</a
+        >
+      </p>
+    </div>
+
+    <script type="module">
+      import { videoPlayerWithProfile } from 'cloudinary-video-player';
+      import 'cloudinary-video-player/cld-video-player.min.css';
+
+      const playerWithDefaultProfile = await videoPlayerWithProfile('player-default-profile', {
+        cloudName: 'demo',
+        profile: 'cldDefault'
+      });
+
+      playerWithDefaultProfile.source('sea_turtle');
+
+      const playerWithCustomProfile = await videoPlayerWithProfile('player-custom-profile', {
+        cloudName: 'dwaq5xqm4',
+        profile: 'https://res.cloudinary.com/dwaq5xqm4/raw/upload/v1/profiles/default_profile.json'
+      });
+
+      playerWithCustomProfile.source('samples/cld-sample-video');
```

Comment:
I do not think we want to document this functionality, we should also remove it. 
```suggestion
```